applications/Makefile

```diff
@@ -8,117 +8,76 @@ AWSMFLAGS= --inline-constant-globals --runtime-globals
 # --whole-archive causes the symbols in the listed static archive to be exported from the resulting *.so
 # https://stackoverflow.com/questions/805555/ld-linker-question-the-whole-archive-option
 CFLAGS=-O3 -flto
-LDFLAGS=-shared -fPIC -Wl,--export-dynamic,--whole-archive -L../libsledge/dist/ -lsledge -Wl,--no-whole-archive -lm
+LDFLAGS=-fPIC -Wl,--export-dynamic,--whole-archive -L../libsledge/dist/ -lsledge -Wl,--no-whole-archive
 # LDFLAGS=-flto -fvisibility=hidden
 
+# Compiletime Components to link into *.so
+SLEDGE_BASE_DIR=../
+SLEDGE_RT_DIR=${SLEDGE_BASE_DIR}/runtime/
+SLEDGE_INSTALL_TARGET=${SLEDGE_RT_DIR}/bin/
```

Comment:
INSTALL_TARGET__DIR_ maybe?

Reply:
Or just SLEDGE_INSTALL_DIR? 
Actually maybe even SLEDGE_EXEC_DIR to be more direct?

Reply:
It's actually unused. I seem to recall that I can't use a variable withing a Make rule's name. Pretty goofy TBH!